advanced/Scripts/setupLCD.sh

```diff
@@ -15,28 +15,28 @@
 # Borrowed from adafruit-pitft-helper < borrowed from raspi-config
```

Comment:
Totally need to re-do this script, Adafruit is bad on this, we can do better. @dschaper